.php_cs

```diff
@@ -86,38 +87,37 @@ $config->setRules(
         'no_unneeded_final_method' => true,
         'no_unused_imports' => true,
         'no_useless_else' => true,
-        'no_useless_return' => true,
```

Comment:
👎

Reply:
Personally, I prefer having such return for symmetry.

Reply:
Not sure what you mean by symmetry here...

I like this rule, because - a good proportion of the time - it's going to make you think about whether the function should be returning something... sane at the end of the method, or if it's void, whether it shouldn't be.

If you've a function where you're returning a value in an if() or similar, but getting to the end of the function and not returning a value, I'd call that a code smell...  If your function returns something, it should return something in all it's code paths... even if it is throwing an exception, or a "Null Object"

Reply:
Don't know what I was thinking of. You're right.